recipes/pycompilation/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pycompilation" %}
+{% set version = "0.4.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycompilation-{{ version }}.tar.gz
+  sha256: fa8095aa956055b37de965d97bec0ba1a29d8b7354cc73bb4094d7286d8b883b
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win or osx]
```

Comment:
Is this package platform dependent at all? It's pure Python, isn't it? Shouldn't it be `noarch: python` then?

Reply:
If it is noarch, then it will attempt to build on win and osx in staged recipes, right? If so, then I can't make this noarch because it doesn't work on the other oses.